packages/shared/src/lib/blocks/block.ts

```diff
@@ -33,20 +33,14 @@ export type PublicBlockPackage = Static<typeof PublicBlockPackage>;
 export type BlockPackage = PrivateBlockPackage | PublicBlockPackage;
 
 export enum BlockCategory {
-  ARTIFICIAL_INTELLIGENCE = 'ARTIFICIAL_INTELLIGENCE',
-  COMMUNICATION = 'COMMUNICATION',
-  COMMERCE = 'COMMERCE',
+  CLOUD = 'CLOUD',
+  COLLABORATION = 'COLLABORATION',
```

Comment:
as we discussed on slack, I think it should be COMMUNICATION like it was originally